app/scripts/modules/core/src/api/ApiService.ts

```diff
@@ -5,18 +5,21 @@ import { SETTINGS } from 'core/config/settings';
 import { ICache } from 'core/cache';
 import { isNil } from 'lodash';
 
+type toStringable = { toString(): string };
```

Comment:
Thinking I'll switch this to a more explicit `type toStringable = boolean | string | number`